src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplGetCaseByUprnTest.java

```diff
@@ -484,13 +484,11 @@ private void verifyCaseDTOContent(
   private void verifyNewAddressEventSent(
       String expectedAddressType,
       String expectedEstabTypeCode,
-      String orgName,
       Integer expectedCapacity,
       CollectionCaseNewAddress newAddress) {
     newAddress.setCaseType(expectedAddressType);
     newAddress.setSurvey(SURVEY_NAME);
     newAddress.setCollectionExerciseId(COLLECTION_EXERCISE_ID);
-    newAddress.setOrganisationName(orgName);
```

Comment:
Not something to fix right now , since you are fitting into the pattern that was there before,  but ....

I'm not very keen on these tests, since they rely heavily on testing in an obscure way , such that you are relying on the mapper too much (to pass through the expected data). If the mapper is wrong in the first place it is possible the tests could pass since the tests would be compatibly incorrect like the code under test. It also makes the tests obscure and less obvious. When we get time we should simplify and improve these tests.

Reply:
Yes I agree - I found it quite difficult to spot which tests needed amending for that reason I think.